bubble-sort.py

```diff
@@ -0,0 +1,27 @@
+# Omplementación del ordenamiento de burbuja
+
+def bubbleSort(arr):
+    n = len(arr)
+
+    # Iterar sobre todos los elementos del arreglo
+    for i in range(n - 1):
+        # range(n) también funciona pero el ciclo se ejecutará una vez extra, no necesario
+
+        # Los últimos i elementos estan ya en su lugar
+        for j in range(0, n - i - 1):
+
+            # iterar el arreglo de 0 a n-i-1
+            # intercambiar si el elemento encontrado es mayor
+            # que el siguiente elemento
+            if arr[j] > arr[j + 1]:
+                arr[j], arr[j + 1] = arr[j + 1], arr[j]
+
+
+# arreglo de prueba
+arr = [64, 34, 25, 12, 22, 11, 90]
```

Comment:
Este ejemplo puede estar mas completo si agregas números **negativos**
```suggestion
arr = [64, 34, 25, 12, 22, 11, 90, -1, 0, -3]
```